src/css/map-view.css

```diff
@@ -1376,12 +1402,10 @@ other class: .ui-slider-range */
   .viewfinder-prediction__content {
     align-items: center;
     background: var(--map-col-content-bkg, var(--map-col-bkg__deprecate));
-    /* TODO(ianguerin) */
-    border: 1px solid var(--map-col-border);
+    border: 1px solid var(--map-col-border, white);
```

Comment:
should this be --map-col-border-muted instead?

Reply:
Done